### PR TITLE
Add news for bump version

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
Adds some missing NEWS items and prepares for patch bump.

I don't think we are breaking any API in this version, and if we want to include some PRs to this patch, it might be good to get some feedback on what we might want to include here.

### CODE REVIEW

**Review by CarloLucibello — APPROVED** [review]

**Comment by DhairyaLGandhi** [comment]
Needs #1840

**Comment by cossio** [comment]
This is ready now?

File: NEWS.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 ## v0.12.9
 * Fixed incorrect output and added GPU compatibility for [AlphaDropout](https://github.com/FluxML/Flux.jl/pull/1781).
 * Add trilinear [Upsample layer](https://github.com/FluxML/Flux.jl/pull/1792).
+* Improved [performance of RNNs](https://github.com/FluxML/Flux.jl/pull/1761)
+* Optimisers now accept an `ϵ` keyword argument
```
Comment (cossio) [inline NEWS.md:7]:
```suggestion
* Optimisers now accept an `ϵ` argument
```

**Comment by darsnack** [comment]
I think this is good to go, but it would be nice to get #1849 in too. It should be done assuming tests clear.

**Comment by cossio** [comment]
@darsnack What's the problem with doing  #1849 in a separate patch release?

**Comment by darsnack** [comment]
No problem, just trying to avoid two patches in quick succession. But I really thought #1849 test would pass this time...let's just go ahead and release without it.

**Comment by cossio** [comment]
> two patches in quick succession

I think there is nothing wrong with this. Could even be good, https://www.oxinabox.net/2019/09/28/Continuous-Delivery-For-Julia-Packages.html.

**Comment by cossio** [comment]
Can we tag a release?

**Comment by darsnack** [comment]
Yes, I am just waiting on the CI to turn green (I know there's no logical reason it shouldn't, but it helps to be paranoid when releasing something to users).

**Comment by cossio** [comment]
You mean the green checkmark on the main repo page? I think it's already green.

**Comment by darsnack** [comment]
Yeah, I just triggered the release

**Comment by cossio** [comment]
Great, thanks

**Pull request merged**